foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/PythonLintingMigration.java

```diff
@@ -0,0 +1,153 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.util.pom.LocationAwareMavenReader;
+import com.boozallen.aissemble.upgrade.util.pom.PomModifications;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.Build;
+import org.apache.maven.model.InputLocation;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.getLocationAnnotatedModel;
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.writeModifications;
+
+/**
+ * This migration disables failing on Python linting for existing projects.
+ */
+public class PythonLintingMigration extends AbstractAissembleMigration {
+    private Plugin getHabushuPlugin(Model model) {
+        List<Plugin> plugins = model.getBuild().getPluginManagement().getPlugins();
+        for (Plugin plugin : plugins) {
+            if ("habushu-maven-plugin".equalsIgnoreCase(plugin.getArtifactId())) {
+                return plugin;
+            }
+        }
+        return null;
+    }
+
+    private Object getConfiguration(Plugin plugin) {
+        Object lintingConfiguration = plugin.getConfiguration();
+        return lintingConfiguration;
+    }
+
+    private boolean hasLinting(Object configuration) {
+        String configurationString = configuration.toString();
+        if (configurationString.contains("sourceFailOnLintErrors") || configurationString.contains("testFailOnLintErrors")) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    protected boolean shouldExecuteOnFile(File file) {
+        Model model = getLocationAnnotatedModel(file);
+        if (!"com.boozallen.aissemble".equalsIgnoreCase(model.getParent().getGroupId())) {
+            return false;
+        }
+        Plugin habushuPlugin = getHabushuPlugin(model);
+        Object configuration;
+
+        if (habushuPlugin == null) {
+            return true;
+        } else {
+            configuration = getConfiguration(habushuPlugin);
```

Comment:
A: Super minor, but I feel like this might be a bit easier to track if you just move the assignment and declaration after the early return.
```java
if( habushuPlugin == null ) {
    return true;
}

Object configuration = getConfiguration(habushuPlugin);
```

Reply:
Modified! Thanks Emily!!